content/en/synthetics/private_locations/_index.md

```diff
@@ -465,7 +485,70 @@ Because Datadog already integrates with Kubernetes and AWS, it is ready-made to
 [1]: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 
 {{% /tab %}}
+{{% tab "Windows" %}}
+
+1. Download the [`datadog-synthetics-worker-<version>.amd64.msi` file][101] and run this file from the machine you want to install the private location on. 
+1. Click **Next** on the welcome page, read the EULA, and accept the terms and conditions. Click **Next**.
+1. Modify where where the application will be installed, or leave the default settings. Click **Next**.
+1. To configure your Windows private location, you can either:
+
+   - Paste and enter a JSON configuration for your Datadog Synthetics Private Location Worker. This file is generated by Datadog when you [create a private location][102].
+
+   {{< img src="synthetics/private_locations/configuration_selector_paste.png" alt="Paste in a JSON configuration in the MSI installer" style="width:80%;" >}}
+
+   - Browse or type a file path to a file containing a JSON configuration for your Datadog Synthetics Private Location Worker. You can type the file path or leave it blank and run `synthetics-pl-worker.exe --config=<PathToYourConfiguration>` in the Windows command-line prompt after the installation is completed.
+  
+   {{< img src="synthetics/private_locations/configuration_selector_browse.png" alt="Browser for a JSON configuration in the MSI installer" style="width:80%;" >}}
 
+1. You can apply the following configuration options:
+
+   Apply firewall rules needed by this program to Windows Firewall
+   : Enable to allow the installer to apply firewall rules on install and remove them on uninstall.
+
+   Apply rules to block reserved IPs in Windows Firewall
+   : Enable to configure blocking rules for Chrome, Firefox, and Edge (if they are installed) and add rules to block reserved IP address ranges outbound in Windows Firewall.
+
+   Enable File Logging
+   : Enable to allow the Synthetics Private Location Worker to log files in the installation directory.
+
+   Log Rotation Days
+   : Specifies how many days logs should be kept before being deleted from the local system.
+
+   Logging Verbosity
+   : Specifies the verbosity of the console and file logging for the Synthetics Private Location Worker. Options for verbosity when performing an unattended `msiexec` installation include `-v` for error, `-vv` for information, and `-vvv` for debug.
+
+1. Click **Next** and **Install** to start the installation process. 
+   
+   {{< img src="synthetics/private_locations/settings.png" alt="Configuration options in the MSI installer" style="width:80%;" >}}
```

Comment:
```suggestion
   {{< img src="synthetics/private_locations/settings.png" alt="Synthetics Private Location Worker wizard, MSI installer. Firewall and log settings are displayed." style="width:80%;" >}}
```

this image might make more sense in the previous step (maybe around line 504) not this one